pom.xml

```diff
@@ -259,7 +259,7 @@
 		<spring-cloud-netflix.version>2.2.0.BUILD-SNAPSHOT</spring-cloud-netflix.version>
 		<spring-cloud-openfeign.version>2.2.0.BUILD-SNAPSHOT
 		</spring-cloud-openfeign.version>
-		<brave.version>5.6.1</brave.version>
+		<brave.version>5.6.4</brave.version>
```

Comment:
```suggestion
		<brave.version>5.6.5</brave.version>
```

Reply:
thx. there were actually 2 places with this version, I later found out. hopefully all good now!